pkgs/by-name/cl/clock-rs/package.nix

```
@@ -6,24 +6,24 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "clock-rs";
-  version = "0.1.215";
+  version = "0.1.216";
 
   src = fetchFromGitHub {
     owner = "Oughie";
     repo = "clock-rs";
     tag = "v${version}";
-    sha256 = "sha256-uDEvJqaaBNRxohYqHE6qfqUF07ynRvGwJKWbYfgPEvg=";
+    sha256 = "sha256-sSofMZThR025O2J4PnOMWVEuBt/+oZ01yDAQAijca0A=";
   };
 
-  cargoHash = "sha256-Zry6mkOUdEgC95Y3U3RCXPJUsmaSoRPlHvUThI92GQU=";
+  cargoHash = "sha256-JwZsvbkErmjeNi/BLu96cO0bzyMCUqCyCFLnHf2cHYc=";
```

```suggestion
  cargoHash = "sha256-7gpD7aAma9Ye7sxqm1wGekcovXxdd11XOMf7fKWZrcA=";
```

If I use `nix-update-script` for this package, it suggest same `cargoHash` which appeared in https://github.com/NixOS/nixpkgs/pull/430159#pullrequestreview-3080926957